src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -731,7 +731,7 @@ public void testGetTypeMap() throws Throwable {
   public void testHolderbility() throws Throwable {
     try (Connection connection = getConnection()) {
       try {
-        connection.setHoldability(0);
+        connection.setHoldability(1);
```

Comment:
Can we use the constant variable here instead of number?

Reply:
I have replaced the number with the ResultSet holdability constant.